src/components/Footer.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <!-- Footer -->
+<footer class="page-footer font-small black-color" color="black">
```

Comment:
what is color supposed to do?

Reply:
Hey yeah this is what I actually texted you about. I wanted to make the footer have a black bar as some form of styling? but it wasn't showing up when I run the application